spaceship/lib/spaceship/connect_api/models/certificate.rb

```diff
@@ -0,0 +1,48 @@
+require_relative '../model'
+module Spaceship
+  class ConnectAPI
+    class Certificate
+      include Spaceship::ConnectAPI::Model
+
+      attr_accessor :certificate_content
+      attr_accessor :display_name
+      attr_accessor :name
+      attr_accessor :platform
+      attr_accessor :serial_number
+      attr_accessor :certificate_type
+
+      attr_mapping({
+        "certificateContent" => "certificate_content",
+        "displayName" => "display_name",
+        "expirationDate" => "expiration_date",
+        "name" => "name",
+        "platform" => "platform",
+        "serialNumber" => "serial_number",
+        "certificateType" => "certificate_type"
+      })
+
+      module CertificateType
```

Comment:
These are all the accept values of that will be coming down from the API so it should be already reflected 😊 https://developer.apple.com/documentation/appstoreconnectapi/certificatetype

Reply:
Ohoh, the new web interface has 9 instead of 7 options:

![image](https://user-images.githubusercontent.com/183673/59462170-68842100-8e23-11e9-9c73-105b806fcfdf.png)

So the actual API used by the website has more features as the one we can use now - not good :/

(Looked into the website API a bit, there the `certificateType ` is an object, for `iOS Development it has a `"platform": "ios"` and `"name": "iOS Development"`, and for `Apple Development` there is no `platform` and `"name": "Development"`. The `certificate` itself has `"certificatePlatform": "ios"` for the old one and nothing for the new type.)

Reply:
Yikes! That 9 versus 7 is not great 😬 I will definitely experiment with all of these options when I make the changes into `cert` 🤔